common/ModIO.F90

```diff
@@ -519,6 +519,76 @@ subroutine ReadWallMesh(fn, wall)
 
   end subroutine ReadWallMesh
 
+
+!*********************************************************************
+  subroutine WriteRBCPlain(fn, rbc)
+    character(*) :: fn
+    type(t_rbc) :: rbc
+
+    open(cell_unit, file=trim(fn), action='write')
+
+    write(cell_unit, *) rbc%nlat0, rbc%nlon0
+    write(cell_unit, *) rbc%nlat, rbc%nlon
+    write(cell_unit, *) rbc%celltype
+    write(cell_unit, *) rbc%x
+
+    close(cell_unit)
+  end subroutine WriteRBCPlain
+
+  subroutine ReadRBCPlain(fn, rbc, xc)
```

Comment:
by plain do you mean healthy?

Reply:
Sorry, no this was bad naming on my part. I just mean that:
WriteRBCPlain takes a cell and a filename, and writes _just_ the cell's plain information (w/o any transformations). Then, ReadRBCPlain can reread in this cell model for a different simulation.